tests/testthat/test-lambda-path.R

```diff
@@ -36,7 +36,7 @@ test_that("lambda paths are computed as in glmnet", {
     pars$y <- switch(pars$family,
                      gaussian = rnorm(n, 10, 2),
                      binomial = rbinom(n, 1, 0.8),
-                     multinomial = rbinom(n, 3, 0.5),
+                     multinomial = rbinom(n, 2, 0.5),
```

Comment:
Any reason for this change? It should be fine either way.

Reply:
Only an attempt to try to make it run faster.